src/main/scripts/src/app/shared/perspectives/documents/post-it/document/post-it-document.component.ts

```diff
@@ -18,19 +18,28 @@
  * -----------------------------------------------------------------------/
  */
 
-import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Component, ElementRef, EventEmitter, HostBinding, Input, OnInit, Output, ViewChild} from '@angular/core';
 
 import {Collection} from '../../../../../core/dto/collection';
 import {Document} from '../../../../../core/dto/document';
 import {Attribute} from '../../../../../core/dto/attribute';
-import {AttributePair} from '../document-attribute';
+import {AttributePair} from '../attribute/attribute-pair';
+import {AttributePropertySelection} from '../attribute/attribute-property-selection';
+import {isString, isUndefined} from 'util';
+import {Popup} from '../../../../../utils/Popup';
 
 @Component({
   selector: 'post-it-document',
   templateUrl: './post-it-document.component.html',
   styleUrls: ['./post-it-document.component.scss']
 })
-export class PostItDocumentComponent {
+export class PostItDocumentComponent implements OnInit {
+
+  @HostBinding('style.border-color')
+  private borderColor: string;
```

Comment:
Do we really want to change border color? In my opinion, it does not look good. There is too much of the same color on the page. I think the icon and collection name in that color is enough.

Reply:
I and Maria liked it.. plus it looks way better on collections, so I added it to documents too, to keep the style same.
I'll ask in a `web-ui`, which one to keep.
![selection_001](https://user-images.githubusercontent.com/15381356/29744948-4c77148c-8aaf-11e7-9338-5e81bdc84769.png)